apps/chat/src/store/publication/publication.reducers.ts

```diff
@@ -71,9 +78,10 @@ export const publicationSlice = createSlice({
     ) => {
       state.publications = state.publications.map((p) =>
         p.url === payload.publication.url
-          ? { ...payload.publication, ...p, uploadStatus: UploadStatus.LOADED }
+          ? { ...p, ...payload.publication, uploadStatus: UploadStatus.LOADED }
```

Comment:
shouldn't use one-letter variables
```suggestion
          ? { ...publication, ...payload.publication, uploadStatus: UploadStatus.LOADED }
```